src/sentry/models/releasecommit.py

```diff
@@ -8,8 +8,8 @@
 class ReleaseCommit(Model):
     __core__ = False
 
-    organization_id = BoundedPositiveIntegerField(db_index=True, null=True)
-    project_id = BoundedPositiveIntegerField(db_index=True)
+    organization_id = BoundedPositiveIntegerField(db_index=True)
```

Comment:
we populated these in migration 0281, so it should be fine. was there something in particular you were worried about?